store/gcworker/gc_worker.go

```diff
@@ -1900,6 +1929,9 @@ func (w *GCWorker) doGCPlacementRules(dr util.DelRangeTask) (err error) {
 			return
 		}
 		physicalTableIDs = append(physicalTableIDs, historyJob.TableID)
+		// Since the original implementation doesn't handle pd rules for ActionDropSchema etc.,
+		// we don't handle TiFlash pd rules for them either.
+		deletePdRuleFunc()
```

Comment:
In `model.ActionDropTablePartition`,  is not necessary here. Since a truncate is just a replacement.

I am not sure if I need to handle `model.ActionDropSchema`, `model.ActionTruncateTablePartition`, and `model.ActionDropTablePartition`.

I think if they are not handled here, then their pd rules will be deleted in HandlePlacementRuleRoutine, which will cause a longer delay before pd rule is gc-ed.

Reply:
This it not how it works. You just delete all rules with id `physicalTableIDs`, and you don't need `HandlePlacementRuleRoutine` at all.